### DISCUSSION TITLE
モバイル学習コンテンツで、OpenAPIドキュメントのリンクはファイル直接のほうが良いか

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#264] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/todo-app/networking/generate-api-client
APIクライアントの作成

OpenAPIドキュメントのリンクは現在OpenAPIドキュメントの格納フォルダが指定されているが、
直接OpenAPIドキュメント（openapi.yaml）でも良い？
フォルダ指定になっている理由を要確認。

・フォルダへのリンク（現状）
https://github.com/ws-4020/mobile-app-hands-on-backend/tree/main/rest-api-specification
・OpenAPIドキュメントへのリンク
https://github.com/ws-4020/mobile-app-hands-on-backend/blob/main/rest-api-specification/openapi.yaml

**Comment by nagano-ryo** [comment]
フォルダでもファイルでもどちらでも問題ないため、対応は保留。

**Comment by creature-water-valley** [comment]
現状フォルダになっているが、それで使いづらいというフィードバックは来ていないクローズする。
